packages/syft/src/syft/service/enclave/enclave_service.py

```diff
@@ -57,8 +57,12 @@ def send_user_code_inputs_to_enclave(
         if isinstance(user_code, SyftError):
             return user_code
 
+        if "reason" in context.extra_kwargs.keys():
+            reason = context.extra_kwargs["reason"]
```

Comment:
Similarly we can replace this with:
reason = context.extra_kwargs.get("reason", ""), where default can empty string.